pkg/uroot/uroot.go

```diff
@@ -22,6 +22,7 @@ var (
 	builders = map[string]Build{
 		"source": SourceBuild,
 		"bb":     BBBuild,
+		"binary": BinaryBuild,
```

Comment:
It would be really useful for nichrome if this were an option when building source? I could use it for NiChrome. I guess that can come later but this is really handy.
so
./u-root -build=source -prebuild=glob

Reply:
I've been thinking that only being able to specify one of these has been holding us back a bit. We should be able to support mixed builds, e.g. being able to say

build 
packages X, Y, Z from source,
packages A, B, C as bb,
and packages D, E, F as standalone binaries.

I'm looking at adding that capability in a different PR.

Reply:
Until then, using the base-initramfs feature to build on top of each other (just like you already do with bb and source builds in NiChrome) should work